db/migrate/20230629124929_create_join_table_cities_packages.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableCitiesPackages < ActiveRecord::Migration[7.0]
+  def change
+    create_join_table :cities, :packages, id: false do |t|
+      # t.index [:city_id, :package_id]
+      # t.index [:package_id, :city_id]
```

Comment:
Any reason for not adding index here..?

Reply:
It was generated by default, when I ran generate migration command. So that's why I didn't change it.